lib/Language/Haskell/Stylish/Config.hs

```diff
@@ -237,8 +235,9 @@ parseLanguagePragmas :: Config -> A.Object -> A.Parser Step
 parseLanguagePragmas config o = LanguagePragmas.step
     <$> pure (configColumns config)
     <*> (o A..:? "style" >>= parseEnum styles LanguagePragmas.Vertical)
-    <*> o A..:? "align" A..!= True
+    <*> o A..:? "align"            A..!= True
     <*> o A..:? "remove_redundant" A..!= True
+    <*> o A..:? "language_prefix"  A..!= "LANGUAGE"
```

Comment:
Can we factor out this line into a separate function that calls `fail` when the prefix is invalid (i.e. not "language")?